### DISCUSSION TITLE
Get Starstruck achievement for transferred repository

### DISCUSSION THREAD

**Discussion by arjunj132** [community/community#42944] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I had this repo with 23 stars on my older account and transferred it to my current account. On my older account, I got the starstruck achievement, and when I transferred it, my new account did not get the achievement while the old account kept the achievement.

So is there any reason for my new account not to get the achievement?

**Comment by radar-blip** [comment]
If you're using the same email for both accounts, I think the achievement would only go to one account. 
You could try changing one of the emails, not sure if something like that would work though.

**Reply by arjunj132** [reply]
I think I had 2 different emails connected to different accounts.

**Reply by knugie** [reply]
FYI, I'm having the same problem and try to solve it... unfortunately changing emails did not work... I removed the email from my old account and added it to my new account. This didn't have any effect: all badges (Arctic Code Vault Contributor, Starstruck, Pull Shark) are stuck with my old account. Any thoughts?

**Comment by jfullstackdev** [comment]
if an email is already used, it cannot be used by other users even if you are the same person using it

all of the badges and commits are tied to an email, so better be consistent with those emails

**Reply by arjunj132** [reply]
Ok, so my new account won't get the badge as it is tied to my old account, thanks! Just asking though, is there any way for my new account to get the badge, or is it stuck in my old account?

**Comment by stjepanmamusa** [comment]
Hi guys, I'm a bit off-topic here, but you seem to know your stuff. Is it possible to get the achievement if you create a repository inside an organization and it get's a certain amount of stars? Or is it only for personal repositories? Thanks a bunch.

**Reply by radar-blip** [reply]
I think only for personal repositories but not 100% sure.

**Reply by arjunj132** [reply]
The project I was talking about was inside an organization managed by my old account, and it got more than 16 stars and the account owning that organization got the achievement.

**Reply by stjepanmamusa** [reply]
@arjunj132 Thank you for the info, that clarifies it 🙂
